src/utils/numAbbr.js

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
This should be a TS file, not using flow anymore.
Likewise its test should also be a TS file.

Reply:
You can see examples in src/components/button or src/components/tooltip. They have mostly tsx files, but yours should just be ts as there is no jsx in here.

Reply:
That was a heck of a lot of work, but it is converted to TS now